cocogen/ccngen/ccn_copy.c

```diff
@@ -10,12 +10,15 @@
 #define MAYBE_UNUSED
 #endif
 
+#pragma GCC diagnostic push
```

Comment:
The warning should already be removed by MAYBE_UNUSED, introduced by eab0984bb999ffd8d7c70825726ba8bbff7b40f0. Are you sure you still get the warning with this change? We also have to make sure the code is compatible with clang and msvc, so I’d rather not have GCC pragmas unless it’s surrounded by define guards (see the other commit)

Reply:
I still get the warnings without the pragma (I checked I do have the MAYBE_UNUSED), I will add the define guard to the pragmas